resource_vpc_peering_connection.go

```diff
@@ -58,15 +70,34 @@ func resourceVPCPeeringConnection() *schema.Resource {
 func resourceVPCPeeringConnectionCreate(d *schema.ResourceData, m interface{}) error {
 	client := m.(*aiven.Client)
 	projectName, vpcID := splitResourceID2(d.Get("vpc_id").(string))
-	pc, err := client.VPCPeeringConnections.Create(
-		projectName,
-		vpcID,
-		aiven.CreateVPCPeeringConnectionRequest{
-			PeerCloudAccount: d.Get("peer_cloud_account").(string),
-			PeerVPC:          d.Get("peer_vpc").(string),
-		},
+	peerRegion := d.Get("peer_region").(string)
+
+	var (
+		pc  *aiven.VPCPeeringConnection
+		err error
 	)
 
+	if peerRegion != "" {
```

Comment:
Only having single `Create` in the API client and something like

```
apiPeerRegion := nil
if peerRegion != "" {
    apiPeerRegion = &peerRegion
}
pc, err := client.VPCPeeringConnections.Create(...)
```

would seem more convenient.